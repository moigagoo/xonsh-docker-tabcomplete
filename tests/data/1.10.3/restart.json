{
        "arguments": [
                "--help",
                "-t", "--time=10"
        ],
        "commands": []
}
