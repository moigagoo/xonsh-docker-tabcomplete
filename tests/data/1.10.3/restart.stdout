
Usage:	docker restart [OPTIONS] CONTAINER [CONTAINER...]

Restart a container

  --help             Print usage
  -t, --time=10      Seconds to wait for stop before killing the container
